### ISSUE TITLE
Use without NVIDIA GPU + CUDA?

### ISSUE THREAD

**Issue by SB2020-eye** [jixiaozhong/RealSR#36]
Hi. I'm just wondering, is there any possibility of using RealSR if I do not have NVIDIA GPU + CUDA?

**Comment by GarenLancaster** [comment]
Yes it can.But you need to change the yml file a little bit.Commet the line gpu_ids to run in cpu mode.
